app/controllers/hubstats/repos_controller.rb

```diff
@@ -46,16 +46,10 @@ def show
     #
     # Returns - the stats for the entirety of Hubstats and all repos
     def dashboard
-      if params[:query] ## For select 2
```

Comment:
It's not necessary here because we have `index` and `dashboard` as different methods. If we just had one, then we would need to keep this part in. Originally having that makes the repo list into the filter on the left side of the page, but the filter is currently using `index`, not `dashboard`, so we can get rid of the part in `dashboard`.